src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/ExceptionRecordMapper.java

```diff
@@ -19,7 +25,24 @@ public ExceptionRecord mapEnvelope(Envelope envelope) {
             envelope.jurisdiction,
             getLocalDateTime(envelope.deliveryDate),
             getLocalDateTime(envelope.openingDate),
-            mapDocuments(envelope.documents)
+            mapDocuments(envelope.documents),
+            mapOcrData(envelope)
         );
     }
+
+    private List<CcdCollectionElement<CcdKeyValue>> mapOcrData(Envelope envelope) {
+        if (envelope.ocrData != null) {
+            return envelope
+                .ocrData
+                .entrySet()
+                .stream()
+                .map(entry ->
+                    new CcdCollectionElement<>(
+                        new CcdKeyValue(entry.getKey(), entry.getValue())
+                    )
+                ).collect(toList());
```

Comment:
Now giving the different constructor of `CcdKeyValue` object, we are able to stream this with no additional indentation :)
```java
return ocrData
    .entrySet()
    .stream()
    .map(CcdKeyValue::new)
    .map(CcdCollectionElement::new)
    .collect(toList());
```